roles/etcd/templates/etcd.conf.j2

```diff
@@ -11,18 +23,20 @@
 # http_read_timeout = 10
 # http_write_timeout = 10
 # key_file = ""
-# peers = []
+{% if etcd_cluster %}
+peers = [ {{ etcd_peers() }}]
+{% endif %}
 # peers_file = ""
 # max_cluster_size = 9
 # max_result_buffer = 1024
 # max_retry_attempts = 3
-# name = "default-name"
+name = "{{ ansible_fqdn }}"
```

Comment:
@sdodson in that case, I would say that we can leverage local_facts and create a new unique identifier if the fact does not exist.  openshift_common, openshift_master and openshift_node have examples of setting local facts for a host.